lib/kennel/syncer.rb

```diff
@@ -214,8 +214,8 @@ def print_diff(diff)
     # We've already validated the desired objects ('generated') in isolation.
     # Now that we have made the plan, we can perform some more validation.
     def validate_plan
-      @update.each do |_, expected, actual, diffs|
-        expected.validate_update!(actual, diffs)
+      @update.each do |_, expected, _, diffs|
+        expected.validate_update!(diffs)
```

Comment:
This feels to me like tailoring a bit too much to the _current_ use case. Providing the `actual` could be useful to the receiver and costs us almost nothing to provide.

Reply:
hmm true, might be useful